shell/platform/windows/flutter_windows_engine.cc

```diff
@@ -523,8 +522,6 @@ bool FlutterWindowsEngine::DispatchSemanticsAction(
 }
 
 void FlutterWindowsEngine::UpdateSemanticsEnabled(bool enabled) {
-  using AccessibilityBridgeDelegateWindows = AccessibilityBridgeDelegateWin32;
```

Comment:
I should probably have cleaned this up when I deleted the #ifdef'ed Win32/UWP; no longer necessary either way.